src/RageSoundReader_Chain.cpp

```diff
@@ -23,7 +23,10 @@
  */
 RageSoundReader_Chain::RageSoundReader_Chain()
 {
-	m_iPreferredSampleRate = 44100;
+	m_iPreferredSampleRate = PREFSMAN->m_iSoundPreferredSampleRate;
+	if (m_iPreferredSampleRate == 0)
+		m_iPreferredSampleRate = 44100; // Default if system default (0) is chosen

```

Comment:
Add braces.

Also The comment here is unnecessary since it's obvious from the code.

Also Also, any reason we're not using the constant here?